feat(plugin-coverage): allow passing results as strings (#499)
code-pushup/cli: packages/plugin-coverage/README.md

The plugin accepts the following parameters:

- `coverageTypes`: An array of types of coverage that you wish to track. Supported values: `function`, `branch`, `line`. Defaults to all available types.
- `reports`: Array of information about files with code coverage results. LCOV format is supported for now.
  - For a single project, providing paths to results as strings is enough.

>   - If you have an Nx monorepo, both path to results (`resultsPath`) and path from the root to project the results belong to (`pathToProject`) need to be provided for the LCOV format. You can adjust our helper function `getNxCoveragePaths` to get the path information automatically.

Reviewer: ```suggestion
  - If you have a monorepo, both path to results (`resultsPath`) and path from the root to project the results belong to (`pathToProject`) need to be provided for the LCOV format. For Nx monorepos, you can use our helper function `getNxCoveragePaths` to get the path information automatically.
```
